ansible/influxdb_restore.yml

```diff
@@ -0,0 +1,9 @@
+---
+
+- name: influxdb restore
+  hosts: influxdb-restore
```

Comment:
please variablize this value, as we don't have to add ips in inventory for this specific task, and it'll be duplicate for influx group which we already have.

Reply:
Hostname is variablised now, we need to pass hostname or ip as jenkins parameter. @rjshrjndrn